pkg/sdn/api/types.go

```diff
@@ -44,7 +44,7 @@ type NetNamespace struct {
 	kapi.ObjectMeta
 
 	NetName string
-	NetID   uint
+	NetID   *uint
```

Comment:
If we change api/v1/types.go, won't this break backward compatibility? (v1 resource populated using 3.1 release will have NetID as uint in the etcd datastore)